GCC5 is now required

WRT Release 0.5.0 testing for ticket #2269

When following the release instructions https://wallaroo-docs-rc.netlify.com/book/getting-started/linux-setup.html as of today/2018-07-18:

```
Install GCC 4.7 or Higher

If you have at least GCC 4.7 installed on your machine, you don't need
to do anything. If you have gcc 4.6 or lower, you'll need to
upgrade. You can check your gcc version by running:
```

My Ubuntu Trusty/14.04 LTS VM has gcc 4.8.4 installed.  However, when trying to compile Machida using the pre-compiled package of ponyc version `0.24.0-79af8a0 [release]`, I get this error.

```
ponyc: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by ponyc)
make: *** [/home/vagrant/wallaroo-tutorial/wallaroo/machida/build/machida] Error 1
```

Installing GCC 5 via the same linux-setup.html guide does indeed allow `ponyc` to run without error and compile Machida.

--- a/book/getting-started/linux-setup.md
+++ b/book/getting-started/linux-setup.md
@@ -55,15 +55,15 @@ sudo apt-get install -y build-essential
 sudo apt-get install -y libssl-dev
 ```
 
-### Install GCC 4.7 or Higher
+### Install GCC 5 or Higher
 
- If you have at least GCC 4.7 installed on your machine, you don't need to do anything. If you have gcc 4.6 or lower, you'll need to upgrade. You can check your `gcc` version by running:
+ If you have at least GCC 5 installed on your machine, you don't need to do anything. If you have gcc 4.9 or lower, you'll need to upgrade. You can check your `gcc` version by running:
 
 ```bash
 gcc --version
 ```
 
-To upgrade to GCC 5 if you don't have at least GCC 4.7:
+To upgrade to GCC 5:
 
 ```bash
 sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test

--- a/book/go/getting-started/linux-setup.md
+++ b/book/go/getting-started/linux-setup.md
@@ -60,15 +60,15 @@ sudo apt-get install -y build-essential
 sudo apt-get install -y libssl-dev
 ```
 
-### Install GCC 4.7 or Higher
+### Install GCC 5 or Higher
 
- If you have at least GCC 4.7 installed on your machine, you don't need to do anything. If you have gcc 4.6 or lower, you'll need to upgrade. You can check your `gcc` version by running:
+ If you have at least GCC 5 installed on your machine, you don't need to do anything. If you have gcc 4.9 or lower, you'll need to upgrade. You can check your `gcc` version by running:
 
 ```bash
 gcc --version
 ```
 
-To upgrade to GCC 5 if you don't have at least GCC 4.7:
+To upgrade to GCC 5:
 
 ```bash
 sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test

